relay-general/src/store/normalize.rs

```diff
@@ -3010,4 +3053,68 @@ mod tests {
         )
         "###);
     }
+
+    #[test]
+    fn test_keeps_valid_measurement() {
+        let name = "lcp";
+        let measurement = Measurement {
+            value: Annotated::new(420.69),
+            unit: Annotated::new(MetricUnit::Duration(DurationUnit::MilliSecond)),
+        };
+
+        let is_dropped = test_is_measurement_dropped(name, measurement);
+        assert_eq!(is_dropped, false);
+    }
+
+    #[test]
+    fn test_drops_too_long_measurement_names() {
+        let name = "lcpppppppppppppppppppppppppppp";
+        let measurement = Measurement {
+            value: Annotated::new(420.69),
+            unit: Annotated::new(MetricUnit::Duration(DurationUnit::MilliSecond)),
+        };
+
+        let is_dropped = test_is_measurement_dropped(name, measurement);
+        assert_eq!(is_dropped, true);
+    }
+
+    #[test]
+    fn test_drops_measurements_with_invalid_characters() {
+        let name = "i æm frøm nørwåy";
+        let measurement = Measurement {
+            value: Annotated::new(420.69),
+            unit: Annotated::new(MetricUnit::Duration(DurationUnit::MilliSecond)),
+        };
+
+        let is_dropped = test_is_measurement_dropped(name, measurement);
+        assert_eq!(is_dropped, true);
+    }
+
+    fn test_is_measurement_dropped(name: &str, measurement: Measurement) -> bool {
```

Comment:
Our actual test cases have the `test_` prefix, so I would omit it for the helper function.

```suggestion
    fn measurement_is_dropped(name: &str, measurement: Measurement) -> bool {
```